File: Soe.Common/DTO/FilesLookupDTO.cs
using SoftOne.Soe.Common.Attributes;
using SoftOne.Soe.Common.Util;
using System.Collections.Generic;

namespace SoftOne.Soe.Common.DTO
{
    [TSInclude]
    public class FilesLookupDTO
    {
        public SoeEntityType Entity { get; set; }
        public List<ImportFileDTO> Files { get; set; }
    }
}

File: Soe.Angular/TypeScript/Common/Models/FilesLookupDTO.ts
import { SoeEntityType } from '../../Util/CommonEnumerations'
import { IFilesLookupDTO } from "../../Scripts/TypeLite.Net4";
import { ImportFileDTO } from './ImportFileDTO';

export class FilesLookupDTO implements IFilesLookupDTO {
  constructor(
    public entity: SoeEntityType,
    public files: ImportFileDTO[]
  ) { }
}
